opendps/Makefile

```diff
@@ -182,6 +183,23 @@ fonts:
 	@python ./gen_lookup.py -f $(METER_FONT_FILE) -s $(METER_FONT_MEDIUM_SIZE) -o meter_medium
 	@python ./gen_lookup.py -f $(METER_FONT_FILE) -s $(METER_FONT_LARGE_SIZE) -o meter_large
 
+graphics:
```

Comment:
If we renamed the logos to have a postfix `cc.gfx.png, logo.gfx.png` we might be able to do something like:
```
%.gfx.o: %.gfx.png
    @python ./gen_lookup.py -i $< -o $@
```

Not sure if Makefiles let you do this with a prefix.

Reply:
I guess the only downside to this is it means the end user _must_ have python and Pillow installed to compile the firmware

Reply:
Unless we check-in the `.gfx.o` files. Then the makefile should not create them unless the user performed a clean / cleanall.

If they use the docker image it should not matter. (Reminder to update that too). If you are going to be building this manually rather than use the releases, I think it could be expected that you have these tools installed.

Reply:
It would of course be handy to have the graphics rebuild when a png is modified but I could live with a manual step for doing that and not require Pillow to be installed for firmware compilation.